raster/colormap-renderer/src/main/java/com/esri/samples/colormap_renderer/ColormapRendererSample.java

```diff
@@ -54,14 +55,18 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // create a raster from a local raster file
       Raster raster = new Raster(new File(System.getProperty("data.dir"), "./samples-data/raster/ShastaBW.tif").getAbsolutePath());
 
       // create a raster layer
       RasterLayer rasterLayer = new RasterLayer(raster);
 
-      // create a Map with imagery basemap
-      ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+      // create a map with an imagery basemap style
```

Comment:
I realise I'm commenting on these to change to `the`, but Im not 100% sure it's correct. My rationale is since it's a unique value from an enum, there can't be any other `imagery basemap style`s, so it's correct to refer to as `the style` rather than `a style`?
```suggestion
      // create a map with the imagery basemap style
```

Reply:
Yeah I think 'a basemap style' but 'the specific basemap style' sounds correct.